src/main/java/com/github/sider/javasee/command/CheckCommand.java

```diff
@@ -47,30 +47,33 @@ public String getName() {
         formatter.onStart();
 
         try {
-            if(!configPath().isFile()) {
+            if(!configPath().isFile() || !configPath().exists()) {
                 err.println("Configuration file " + configPath() + " does not look a file.");
                 err.println("Specify configuration file by -config option.");
-                return JavaSee.ExitStatus.CONFIG_FILE_UNKNOWN_ERROR;
+                return JavaSee.ExitStatus.CONFIG_FILE_NOT_FOUND;
             }
             var rootPath = Optional.ofNullable(optionRoot).map((root) -> new File(root)).orElse(configPath().getParentFile());
             Map<String, Object> yaml;
             Config config = null;
             try {
                 yaml = new Yaml().load(new FileInputStream(configPath()));
                 if(yaml == null) {
-                    System.out.println("YAML file has unknown error");
+                    System.out.println("YAML file \"" + configPath() + "\" has unknown error");
                     return JavaSee.ExitStatus.CONFIG_FILE_UNKNOWN_ERROR;
                 }
-                config = Config.load(yaml, configPath(), rootPath);
+                config = Config.load(configPath(), rootPath);
             } catch (Exceptions.YamlValidationException e) {
-                System.out.println("YAML file has schema error: " + e.getMessage());
+                System.out.println("YAML file \"" + e.configPath + "\" has schema error: " + e.getMessage());
                 return JavaSee.ExitStatus.CONFIG_FILE_SCHEMA_ERROR;
             } catch (FileNotFoundException e) {
                 System.err.println("YAML file is not found: " + e.getMessage());
                 return JavaSee.ExitStatus.CONFIG_FILE_NOT_FOUND;
             } catch (YAMLException e) {
-                System.out.println("YAML file has syntax error: " + e.getMessage());
+                System.err.println("YAML file has syntax error: " + e.getMessage());
```

Comment:
[suggestion] Can we add a file path also here?

```suggestion
                System.err.println("YAML file \"" + configPath() + "\" has syntax error: " + e.getMessage());
```